jwql/website/apps/jwql/data_containers.py

```diff
@@ -896,14 +901,14 @@ def thumbnails_ajax(inst, proposal=None):
             filename_dict = filename_parser(rootname)
         except ValueError:
             # Temporary workaround for noncompliant files in filesystem
-            filename_dict = {'activity': file_id[17:19],
-                             'detector': file_id[26:],
-                             'exposure_id': file_id[20:25],
-                             'observation': file_id[7:10],
-                             'parallel_seq_id': file_id[16],
-                             'program_id': file_id[2:7],
-                             'visit': file_id[10:13],
-                             'visit_group': file_id[14:16]}
+            filename_dict = {'activity': rootname[17:19],
```

Comment:
@laurenmarietta Good call, `rootname` is a more consistent name to use here rather than `file_id`.